src/earn/anchor/AnchorEarnProvider.ts

```diff
@@ -0,0 +1,54 @@
+import { AnchorEarn, CHAINS, DENOMS, NETWORKS, OperationError, Output } from '@anchor-protocol/anchor-earn';
+import store from '../../store';
+import { EarnProvider } from '../EarnProvider';
+
+class AnchorEarnProvider extends EarnProvider {
+  client: AnchorEarn;
+
+  constructor({ network }: { network: string }) {
+    super();
+
+    this.client = new AnchorEarn({
+      chain: CHAINS.TERRA,
+      network: network === 'mainnet' ? NETWORKS.COLUMBUS_5 : NETWORKS.BOMBAY_12,
+      mnemonic: store.state.wallets[0].mnemonic,
```

Comment:
hmm this might be a problem, what about the account?

Reply:
What do you mean by "account"? Aren't they based on the same mnemonic?

Reply:
They are based on the mnemonic, but the question is, if a user has 3 terra accounts, which is used? The account that is used for the anchor earning should be user specified

Reply:
Hmm trying to wrap my head around 😆 . Do we really care about which account is used because the AnchorClient is created by the mnemonic and not public address OR you count as a account `store.state.wallets[1].mnemonic` and then `store.state.wallets[2].mnemonic`  OR i do get this completely wrong :( .

Reply:
A user's wallet has multiple accounts. On each account, the funds are separate right? When you want to stake, which account will be used? Here we are only specifying the mnemonic. What if we have 3 accounts, each with 10 UST. Which is used?

Reply:
gotcha, no idea right now later when we integrate it into the wallet i guess we can understand and adjust it cuz really not sure rn